src/App.css

```diff
@@ -3,79 +3,338 @@
   padding: 0;
   list-style: none;
   text-decoration: none;
+  font-family: 'Montserrat', sans-serif;
 }
 
-.header {
-  padding: 10px;
-  margin: 10px;
+.bookcard {
+  padding-top: 10px;
+  background: #fff;
+  margin: 10px auto;
+  width: 90%;
+  height: 150px;
+  display: flex;
+  justify-content: space-between;
```

Comment:
- [ ] Great job so far, you are almost there. Please remember we are reproducing the Zeplin design, be it color, spacing and positioning. Kindly ensure the book's title have a black font color like in the original design.
- [ ] please reduce the white gap behind the update progress button, so it can align well with the original design.
yours
![Screenshot_40](https://github.com/enocol/Bookstore/assets/65298048/2cbb9194-2407-4cdb-b533-ac0b5f105b4e)
original
![Screenshot_37](https://github.com/enocol/Bookstore/assets/65298048/a0b1a285-bfb0-4055-9847-cb969fb577fb)